extensions/DynamicPageList/DPLSetup.php

```diff
@@ -1550,7 +1550,7 @@ public static function endReset( &$parser, $text ) {
         return true;
     }
 
-    public static function endEliminate( &$parser, &$text ) {
+    public static function endEliminate( $parser, &$text ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$text". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)